gke.tf

```diff
@@ -20,6 +20,8 @@ resource "google_container_cluster" "domino_cluster" {
   # node pool and immediately delete it.
   remove_default_node_pool = true
 
+  # On version 5.0.0+ of the provider, you must explicitly set deletion_protection = false
+  # and run terraform apply to write the field to state in order to destroy a cluster.
   deletion_protection = false
```

Comment:
This isn't for this PR, but I wonder if this should default to true and be set to false by the deployer (conditionally). You'd think this isn't the default we'd want for production, and if someone uses this for production and accidentally tries to destroy...